hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAsyncTableRegionReplicasScan.java

```diff
@@ -39,7 +39,7 @@
   public static final HBaseClassTestRule CLASS_RULE =
     HBaseClassTestRule.forClass(TestAsyncTableRegionReplicasScan.class);
 
-  private static int ROW_COUNT = 1000;
+  private static int ROW_COUNT = 100;
```

Comment:
What's the rationale for reducing row counts for the Async tests? Decrease runtime?

Reply:
This one is failing on me mysteriously, exiting w/o dumping logs... just broke xml files. This desperate attempt at seeing if it would improve. Let me just put back the count to 1000 till I figure real issue.